src/pages/haesoo/Main/MainLeft/Feed/Comment/Comment.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import AddComment from './AddComment/AddComment';
+
+export default function Comment() {
+  const [commentData, setCommentData] = useState([]);
+  const [commentInput, setCommentInput] = useState('');
+  const [comments, setComment] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/haesoo/commentData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setCommentData(data);
+      });
+  }, []);
+
+  const handleChange = event => {
+    const { value } = event.target;
+    setCommentInput(value);
+  };
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    setComment(
+      comments.concat({
+        id: '',
+        userName: 'haesoo',
+        content: commentInput,
+        isLiked: '',
+      })
+    );
+    setCommentInput('');
+  };
```

Comment:
> 가독성 향상을 위해! 

concat 안에 배열의 줄 수가 긴데, 한줄로 줄여보는건 어떠신가요??
```suggestion
 const newComment = {
   id: '',
   어쩌구저쩌구
   ...
 }
 
 const handleSubmit = event =>{
   event.preventDefault(); 
   setComment(newComment, [...comments]);
   setCommentInput(''); 
 }
```